assets/.jshintrc

```diff
@@ -3,25 +3,33 @@
   "browser": true,
   "esnext": true,
   "bitwise": true,
-  "camelcase": true,
+  "camelcase": false,
   "curly": true,
   "eqeqeq": true,
   "immed": true,
   "indent": 2,
   "latedef": true,
-  "newcap": true,
+  "newcap": false,
   "noarg": true,
+  "quotmark": false,
   "regexp": true,
-  "undef": true,
-  "unused": true,
+  "smarttabs": true,
   "strict": true,
+  "sub" : true,
   "trailing": true,
-  "smarttabs": true,
+  "undef": true,
+  "unused": true,
   "globals": {
     "angular": false,
-    "$": false,
-    "URI": false,
+    "hawtioPluginLoader": false,
+    "HawtioCore": false,
+    "Logger" : false,
+    "LabelSelector": false,
     "moment": false,
-    "hawtioPluginLoader": false
+    "Messenger" : false,
+    "URI": false,
+    "ZeroClipboard": false,
+    "$": false,
+    "_" : false
   }
```

Comment:
This is less messy than it looks, alphabetised & compared to the /tests/.jshint file to ensure consistency.